benchmarks/random_allocations/random_allocations.cpp

```diff
@@ -62,7 +62,7 @@ void random_allocation_free(rmm::mr::device_memory_resource& mr,
                             SizeDistribution size_distribution,
                             size_t num_allocations,
                             size_t max_usage,  // in MiB
-                            rmm::cuda_stream_view stream = {})
+                            rmm::cuda_stream_view const& stream = {})
```

Comment:
It is more canonical to pass view types by value. `column_view` is an exception to the rule.

Reply:
Damn, that was a big change...

What's the problem with a const ref?

Reply:
There's nothing really _wrong_ with pass by const ref, it's just the norm to pass by value. Same as how when you pass an `int` to a function it's usually done by value and not `int const&`.

Reply:
find-replace ftw.

Reply:
Done.